app/services/declarations/create.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+module Declarations
+  class Create
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attribute :lead_provider
+    attribute :participant_id
+    attribute :declaration_type, :string
+    attribute :declaration_date, :datetime
+    attribute :course_identifier, :string
+    attribute :has_passed
```

Comment:
I expect the answer is yes, but just checking we're omitting `evidence_held` intentionally here?

Reply:
Hi @ethax-ross think `evidence_held` is only applicable to declarations for ECF profiles isn't it?